sync/pacman/pacman-script

```diff
@@ -15,9 +15,12 @@ REPO_DIR="${1}"
 YAMLFILE="${1}/pacman.yaml"
 TEMPORARY_DIR="${REPO_DIR}/.tmp"
 
+# import functions from pacman-utils script
+source "${REPO_DIR}/sync/pacman/pacman-utils"
+
 echo "Make sure temporary dir is created and empty (${TEMPORARY_DIR})"
-rm -rf "${TEMPORARY_DIR}"
-mkdir -p "$TEMPORARY_DIR"
+rm -rf $TEMPORARY_DIR
```

Comment:
this probably doesn't want to change; this would make accidentally destroying one's filesystem far easier. e.g if REPO_DIR contained a space